utils/remote.c

```diff
@@ -645,6 +647,22 @@ static void addDays(tr_variant* args, tr_quark const key, char const* arg)
     }
 }
 
+static void addLabels(tr_variant* args, const tr_quark key, const char* arg)
+{
+    tr_variant* labels = tr_variantDictAddList(args, key, 10);
+    char* argcpy = tr_strdup(arg);
+    char* const tmp = argcpy; /* save copied string start pointer to free later */
+    char* token;
+    while ((token = tr_strsep(&argcpy, ",")) != NULL)
+    {
+        if (*token != '\0')
```

Comment:
First strip, then check if it's empty. We don't want labels consisting entirely of whitespace chars.